.gitpod.yml

```diff
@@ -0,0 +1,36 @@
+vscode:
+  extensions:
+    - twxs.cmake@0.0.17:9s7m9CWOr6i6NZ7CNNF4kw==
+    - ms-vscode.cmake-tools@1.3.1:Yde58UUl5J9XpLmXM+Bqiw==
+    - ms-vscode.cpptools@0.26.2:Pq/tmf2WN3SanVzB4xZc1g==
+    - eamodio.gitlens@10.2.1:ZI2Sl3DHdJu4aG2wW92CQQ==
+
+tasks:
+  - init: |
+      git submodule update --init --recursive
+      mkdir build
+      cmake -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -H/workspace/arbor -B/workspace/arbor/build \
+      -DCMAKE_INSTALL_PREFIX:STRING=/workspace/arbor/install -DCMAKE_BUILD_TYPE:STRING=Debug -G "Unix Makefiles" \
+      -DARB_WITH_PYTHON:BOOL=ON
+      ln -s /workspace/arbor/build/compile_commands.json /workspace/arbor
+    prebuild: |
+      cmake --build /workspace/arbor/build/ --parallel $(nproc)
```

Comment:
Can we make this also `make install`, so that Arbor is installed in the `CMAKE_INSTALL_PREFIX`?

Then we can set the folloing path when the environment starts:
```
export PYTHONPATH=/workspace/arbor/install/lib/python3.7/site-packages
```
So users can start the python module straight away.

Reply:
done